### PR TITLE
Run e2e on ironic-image repo

### PR DESCRIPTION
**Pull request by mboukhalfa**
This PR adds e2e integration test triggers to prow config under ironic-image repo

### CODE REVIEW

**Review by Sunnatillo — COMMENTED** [review]
/lgtm

File: prow/manifests/overlays/metal3/config.yaml [hunk 1]
```diff
@@ -2951,7 +2951,39 @@ presubmits:
                 value: "TRUE"
             image: docker.io/pipelinecomponents/markdownlint-cli2:0.9.0@sha256:71370df6c967bae548b0bfd0ae313ddf44bfad87da76f88180eff55c6264098c
             imagePullPolicy: Always
-
+    # name: {job_prefix}-{image_os}-e2e-integration-test-{capm3_target_branch}
```
Comment (tuminoid) [inline prow/manifests/overlays/metal3/config.yaml:2954]: This is adding tests to mariadb-image starting from L2925?
/hold

**Comment by tuminoid** [comment]
Is this good to go now @mboukhalfa ? If it is, remove the hold.
/cc @lentzi90

**Pull request merged**